docs/src/fs_event.rst

```diff
@@ -55,7 +54,6 @@ Data types
             * or kqueue to detect events. This may not work on remote filesystems such
             * as NFS mounts. This flag makes fs_event fall back to calling stat() on a
             * regular interval.
-            * This flag is currently not implemented yet on any backend.
```

Comment:
I thought it belongs on the function description, it's still not
implemented.
On Jan 16, 2015 7:46 PM, "Ben Noordhuis" <notifications@github.com> wrote:

> In docs/src/fs_event.rst
> <https://github.com/libuv/libuv/pull/135#discussion-diff-23100303>:
>
> > @@ -55,7 +54,6 @@ Data types
> >              * or kqueue to detect events. This may not work on remote filesystems such
> >              * as NFS mounts. This flag makes fs_event fall back to calling stat() on a
> >              * regular interval.
> > -            * This flag is currently not implemented yet on any backend.
>
> What was the reason for removing this? It's still true that they aren't
> implemented, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libuv/libuv/pull/135/files#r23100303>.
>